[2023-03-29 15:32:04,782] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.start_task manual__2023-03-29T15:32:03.946758+00:00 [queued]>
[2023-03-29 15:32:04,791] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: data_merge_dag.start_task manual__2023-03-29T15:32:03.946758+00:00 [queued]>
[2023-03-29 15:32:04,792] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 15:32:04,792] {taskinstance.py:1357} INFO - Starting attempt 1 of 4
[2023-03-29 15:32:04,792] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-03-29 15:32:04,805] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): start_task> on 2023-03-29 15:32:03.946758+00:00
[2023-03-29 15:32:04,810] {standard_task_runner.py:52} INFO - Started process 149 to run task
[2023-03-29 15:32:04,811] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_merge_dag', 'start_task', 'manual__2023-03-29T15:32:03.946758+00:00', '--job-id', '462', '--raw', '--subdir', 'DAGS_FOLDER/data_merge.py', '--cfg-path', '/tmp/tmpk1wcftue', '--error-file', '/tmp/tmpuvysr8em']
[2023-03-29 15:32:04,812] {standard_task_runner.py:80} INFO - Job 462: Subtask start_task
[2023-03-29 15:32:04,859] {task_command.py:369} INFO - Running <TaskInstance: data_merge_dag.start_task manual__2023-03-29T15:32:03.946758+00:00 [running]> on host cbb3159fbb01
[2023-03-29 15:32:04,927] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=kartaca
AIRFLOW_CTX_DAG_ID=data_merge_dag
AIRFLOW_CTX_TASK_ID=start_task
AIRFLOW_CTX_EXECUTION_DATE=2023-03-29T15:32:03.946758+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-29T15:32:03.946758+00:00
[2023-03-29 15:32:04,928] {logging_mixin.py:115} INFO - DAG has been started...
[2023-03-29 15:32:04,928] {python.py:173} INFO - Done. Returned value was: DAG has been started...
[2023-03-29 15:32:04,956] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=data_merge_dag, task_id=start_task, execution_date=20230329T153203, start_date=20230329T153204, end_date=20230329T153204
[2023-03-29 15:32:04,984] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-03-29 15:32:05,022] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
